Sofware Management Plan/README.md

```diff
@@ -0,0 +1,137 @@
+# Software Management Plan
+
+A sofware management plan should be provided for all projects which contribute to the ECMWF software stack.
+This is to ensure that contributions are appropriate, visible, follow guidelines and there is a clear roadmap
+for development and ownership.
+
+## 1. Project Overview
+
+Provide a concise overview to allow others to understand the project and its purpose.
+
+Include:
+
+- **Project name**
+- **Purpose and goals**
+- **Problem it solves / value it provides**
+- **Scope and boundaries** (what is included and excluded)
+- **Relationship to the ECMWF software stack**  
+  - New components?
+  - Extension?
+  - Replacement of legacy functionality?
+
+---
+
+## 2. Alignment With ECMWF Strategy and Architecture
+
+Explain how the software fits into ECMWF's broader technical and organisational goals.
+
+Address:
+
+- **Alignment with ECMWF technical strategy**
+- **Fit with ECMWF architecture, APIs, data formats, workflows, and coding standards**
+- **Dependencies** (internal and external)
```

Comment:
I wonder what sort of thing we would write for 'dependencies' - we can't list all the dependencies, as some are optional and they change over time; presumably we'd like this document to be fairly stable.

Reply:
Agreed, it's probably too broad. I think something that indicates that it's dependancies we're interested in, e.g. internal dependancies or dependencies on less mature external software (maybe this example belongs in risk assessment?)